src/libraries/System.Formats.Tar/tests/TarReader/TarReader.File.Tests.cs

```diff
@@ -241,6 +241,14 @@ public void Throw_SingleExtendedAttributesEntryWithNoActualEntry()
             Assert.Throws<EndOfStreamException>(() => reader.GetNextEntry());
         }
 
+        [Fact]
+        public void ReadDataStreamOfGoLangTarTarGzGnu()
+        {
+            using MemoryStream archiveStream = GetTarMemoryStream(CompressionMethod.GZip, "golang_tar", "pax-bad-hdr-large");
```

Comment:
I assume you are going to use `podman-hello-world.tar` added in https://github.com/dotnet/runtime-assets/pull/280 in a separate PR (the test would fail now?)?